### ISSUE TITLE
get_market_ohlcv_by_date 지수는 return이 없음

### ISSUE THREAD

**Issue by jmche** [sharebook-kr/pykrx#158]
stock.get_market_ohlcv_by_date("20200118", "20230703", 'kosdaq', adjusted=True)    return값이 없습니다.
지수가 아닌 주식코드는 결과가 있는데, 지수로는 결과가 없습니다. 이전버전에는 작동되였는데, 새로운 버전에서 작독되지 않습니다.

이전버전에서는 adjusted가 있으면 naver를 이용한것으로 확인되였지만, 새로운 버전에서는 krx를 사용하고 있습니다.
지수의 결과값도 받을수 있으면 좋을것 같습니다.

**Comment by mr-yoo** [comment]
1)
KRX가 개편됨에 따라 KRX에서도 수정주가의 조회가 가능하게 됐습니다.  웹사이트 개편이후 KRX에서만 데이터 조회하도록 수정했습니다.

![image](https://github.com/sharebook-kr/pykrx/assets/29349211/e2196007-6c7d-4943-9193-834b82fba52d)


2) 
최근 2년 이상 조회하지 못하게 웹서버가 막음에 따라 위 코드가 동작하지 않습니다. 이슈를 해결하려고 준비중입니다.

**Comment by mr-yoo** [comment]
세 번째 파라미터가 잘못됐습니다. `kosdaq` 대신 티커를 입력해야 합니다.
```python
stock.get_market_ohlcv_by_date("20200118", "20230703", 'kosdaq', adjusted=True)
```

**Comment by jmche** [comment]
주식코드같은 경우 2년이상도 조회되는것 같습니다. 
하지만 지수가 조회되지 않습니다. 제거 받고 싶은 데이터는 지수의 데이티입니다. 
혹시 지수데이터를 받을수 있다면 예시로 주시면 감사하겠습니다.

![1688689149318](https://github.com/sharebook-kr/pykrx/assets/10260136/8c668582-38a1-42f1-866f-101212efe8ee)

**Comment by mr-yoo** [comment]
지수는 index_ohlcv 함수를 사용해 주세요.

```python
df = stock.get_index_ohlcv("20190101", "20190228", "1028")
print(df.head(2))
```

티커 목록은 가칭이지만, 다음 형태로 조회할 수 있습니다.
```
tickers = stock.get_index_ticker_list()
print(tickers)
```

**Comment by jmche** [comment]
네 확인해봤습니다. 문제 없이 잘 실행됩니다. 다만 ticker list와 이름을 같이 결과가 보이면 더 좋을것 같은데, 
즉 get_index_ticker_list시 이름도 같이 결과로 나오면 다시한번 get_index_ticker_name을 실행할 필요가 없을것 같습니다.
감사합니다.
